webroot/.htaccess

```diff
@@ -2,4 +2,6 @@
     RewriteEngine On
     RewriteCond %{REQUEST_FILENAME} !-f
     RewriteRule ^ index.php [L]
+    RewriteCond %{HTTP:Authorization} ^(.*)
```

Comment:
I ll just say what is good, because the whole PR makes no sense to me.

This RewriteCond is good!